docs/src/languages.md

```diff
@@ -40,3 +40,10 @@ dotnet add package PlaywrightSharp
 
 * [Playwright on NuGet](https://www.nuget.org/packages/PlaywrightSharp/)
 * [GitHub repo](https://github.com/microsoft/playwright-sharp)
+
+## Java
+
+Playwright for Java is available in preview.
+
+* [Playwright on Maven Central](https://search.maven.org/artifact/com.microsoft.playwright/playwright)
```

Comment:
Let's remove link to the Maven Central for now as it will show 0.172.3 version at the top but it was published by mistake

Reply:
Done.